circle.yml

```diff
@@ -22,6 +21,6 @@ deployment:
     commands:
       - echo $gcloud_json_key_base64 | sed 's/ //g' | base64 -d > /tmp/gcloud_key.json
       - curl https://sdk.cloud.google.com | bash
+      - docker tag codeclimate/codeclimate $PRIVATE_REGISTRY/$CIRCLE_PROJECT_REPONAME:b$CIRCLE_BUILD_NUM
       - gcloud auth activate-service-account --key-file /tmp/gcloud_key.json
-      - gcloud docker -a
-      - docker push $PRIVATE_REGISTRY/$CIRCLE_PROJECT_REPONAME:b$CIRCLE_BUILD_NUM
+      - gcloud docker push $PRIVATE_REGISTRY/$CIRCLE_PROJECT_REPONAME:b$CIRCLE_BUILD_NUM
```

Comment:
I'm not sure either. We [push to docker hub](https://github.com/codeclimate/codeclimate/blob/master/bin/release#L21) as part of the `bin/release` script, which is [pulled during installation](https://github.com/codeclimate/codeclimate/blob/76981f8d17c9184948ff8344dc6d70def359ca93/Makefile#L10).

As far as I know, we just install codeclimate as a gem in builder.